File: metric_tool/test/resources/kotlin/cloc/ClocA.kt
package io.company.app.view.basket.kotlin.cloc

import io.company.app.view.basket.kotlin.cloc.ClocA

/**
 * MyClass
 * this is some comment
 */
internal class ClocA {
    // some comment
    private val myString: String = ""

    /// this returns the local string + the value
    @Override
    private fun myFun(value: String): String {
        val local = myString
        /*
            some multiline
            comment
         */
        return local + value
    }
}

File: metric_tool/test/resources/dart/cloc/cloc_a.dart
library mylib;

import 'package:flutter/material.dart';

/**
 * MyClass
 * this is some comment
 */
class ClocA {
  // some comment
  final String myString = "myString";

  /// this returns the local string + the value
  @override
  String myFun(String value) {
    final local = myString;
    /*
        some multiline
        comment
     */
    return local + value;
  }
}
